### ISSUE TITLE
Dropdowns don't open in desktop view

### ISSUE THREAD

**Issue by winmodify** [iamphill/Bootstrap-Offcanvas#60]
Dropdown menu's don't open when you click on them in desktop view.
In mobile view it works perfectly.

**Comment by vadim-p** [comment]
Same issue. 
It conflicts with bootstrap's dropdown because of `e.stopPropagation()` call in `OffcanvasDropdown. _clickEvent` method.
And I wondered why that class exists at all :)

**Comment by fstockel** [comment]
I can into the same issue and managed to solve it (or make it work) by explicitly adding a display attribute in my css declaration for the shown class on the dropdown-menu:

.navbar-offcanvas .dropdown-menu.shown {
	display:inline-block;
}

**Comment by ChrisInv** [comment]
Thanks for this @fstockel. Problem is the dropdown won't close when clicking away. And if you have multiple dropdowns, then clicking on one won't close the other if it's already open.

Anyone got a solution for this scenario?

**Comment by jer0dh** [comment]
@ChrisInv the workaround provided by @fstockel worked for me.  My dropdowns close when I click on another dropdown.

**Comment by scrapewww** [comment]
This works for me, although clicking outside the menu does not close the menu which is kind of sketchy imo.

**Comment by simpletheoryco** [comment]
@ChrisInv @scrapewww I'm having the same issue of dropdowns not closing when clicking away (outside the navbar, anywhere else on the page). Have you guys figured this out?

**Comment by simpletheoryco** [comment]
Nevermind, I found a fix. Stefan's answer worked for me.

https://stackoverflow.com/questions/29918384/jquery-hide-a-particular-div-until-another-is-clicked-and-hide-that-div-again

**Comment by balajidharma** [comment]
Try below fix - Created pull request #74 

```
         e.stopPropagation();
          $('.dropdown-toggle').not(this.element).closest('.active').removeClass('active').find('.dropdown-menu').removeClass('shown');
          this.dropdown.toggleClass("shown");
 -        return this.element.parent().toggleClass('active');
 +        return this.element.parent().toggleClass('open');
        };
```

**Comment by iamphill** [comment]
https://github.com/iamphill/Bootstrap-Offcanvas/pull/75 was merged
